### ISSUE TITLE
Open files after generating

### ISSUE THREAD

**Issue by aronduby** [dydx/Laravel-5-Artisan#1]
I'm loving this package, so thank you for making it. Only thing I would hope for as an improvement would be auto opening generated files. Would make dealing with migrations nicer.

**Comment by dydx** [comment]
Hey, thanks for using this package! To address your idea, I'm on the fence about adding in something like that. I could see the utility in that, though I also feel like we can achieve a fairly efficient workflow by just knowing the structure of Laravel. For example,
- if I create a new migration titled `create_products_table_migration`, I can quickly use Ctrl-P to fuzzy search for that name.
- if I create a new model called Product, I can also quickly use Ctrl-P to fuzzy search for that name.
- In the case of the Make Migration: Schema commands, I am creating multiple files; knowing how the model is going to be named and what I named my migration, I can again use Ctrl-P to search for those files.

Something I could see being somewhat troubling would be deciding focus on files, and which order to display new files in the case of multiple files being made (such as with Make Migration: Schema).

I'll play with this idea for a bit and see if its something I could feel confident in and that doesn't "get in the way".

It might be worth making that into a separate package and just back in integration, if I decided it was worth while. That way other code generating utilities could plug into the API and work well. 

At any rate, if you have any input on those workflow decisions of issues I mentioned, feel free to chime in. I'm generally open to ideas.
